extensions/content-manager/admin/src/containers/ListView/index.js

```diff
@@ -70,8 +70,29 @@ function ListView({
   const getDataRef = useRef();
   const [isLabelPickerOpen, setLabelPickerState] = useState(false);
   const [isFilterPickerOpen, setFilterPickerState] = useState(false);
+  const [relationApiData, setRelationApiData] = useState([])
   const [idToDelete, setIdToDelete] = useState(null);
   const contentTypePath = [slug, 'contentType'];
+  const relation = get(layouts, [slug, 'contentType', 'settings', 'relationName']);
+  const collectionName = get(layouts, [slug, 'contentType', 'schema', 'collectionName']);
+  const isRelationViewEnabled  = layouts[slug].contentType.settings.relation;
```

Comment:
1. once You are using lodash `get` (2 line above) , and now You are accessing property with traditional way, 
I will reccomend to use lodash `get` here as well for more safety
2. I don't know what type of `relation` field is, but `is` prefix suggest to be boolean, and you probably have string? (i guess)

Reply:
fixed with get. isRelationViewEnabled is a boolean.